WORKSPACE

```diff
@@ -13,19 +13,19 @@ http_archive(
 
 http_archive(
     name = "com_google_protobuf",
-    sha256 = "77ad26d3f65222fd96ccc18b055632b0bfedf295cb748b712a98ba1ac0b704b2",
-    strip_prefix = "protobuf-3.17.3",
+    sha256 = "ba0650be1b169d24908eeddbe6107f011d8df0da5b1a5a4449a913b10e578faf",
+    strip_prefix = "protobuf-3.19.4",
```

Comment:
Could you update the protobuf version in the following docker file as well?

https://github.com/grpc/grpc-web/blob/78313a239aaa5ae039b0420d5e61a18de0c93ee2/net/grpc/gateway/docker/prereqs/Dockerfile#L25

Reply:
Done